python/lsst/pipe/tasks/hips.py

```diff
@@ -1549,19 +1575,19 @@ def _write_allsky_file(self, hips_base_path, allsky_order):
             matches = re.match(pixel_regex, png_uri.basename())
             pix_num = int(matches.group(1))
             tile_image = Image.open(io.BytesIO(png_uri.read()))
-            row = math.floor(pix_num//n_tiles_per_side)
-            column = pix_num % n_tiles_per_side
+            row = math.floor(pix_num//n_tiles_wide)
+            column = pix_num % n_tiles_wide
```

Comment:
Row and column both use the width?

Reply:
Yeah, it's div and mod to get x/y from an ordered list.